app/ts/ManualInput/ManualInputService/ManualInputService.ts

```diff
@@ -227,3 +233,58 @@ export function normalizeInputValue(
 export function renderInputEffect(effect: Effect, usePercentage: boolean) {
   return renderEnteredValues(effect, usePercentage, true);
 }
+
+export function normalizeCells<T extends Effect | Distribution>(
+  datasourceId: string,
+  items: Record<string, Record<string, T>>
```

Comment:
Can it be: 
Record<string, Record<string, Effect|Distribution>>
? If so, I'd prefer that.